src/shared/components/Terms/index.jsx

```diff
@@ -231,13 +231,7 @@ export default class Terms extends React.Component {
                           details={details}
                           docuSignUrl={docuSignUrl}
                           getDocuSignUrl={getDocuSignUrl}
-                          agreeTerm={agreeTerm}
```

Comment:
@zjuasmn I don't really understand, what for have you touched this file, and why have you removed these props? It breaks the terms agreement flow in a modal dialog. Well, I just add these back to the code myself before merging; but if there is some reason for this that I have missed, please comment.

Reply:
@birdofpreyru  Because My IDE told me that this props are redundant. I check the implement of `component/Terms/TermDetails` and found that the propTypes of TermDetails only requires `details`, `docuSignUrl`, `loadingDocuSignUrl` and `getDocuSignUrl` and the methods in it didn't contains code related to other props.

Reply:
@zjuasmn oh, I've double-checked, and you are right. I did some mistake when testing originally.